src/test/java/redis/clients/jedis/tests/JedisSentinelPoolTest.java

```diff
@@ -187,7 +186,7 @@ private void forceFailover(JedisSentinelPool pool) throws InterruptedException {
 
     Jedis afterFailoverJedis = pool.getResource();
     assertEquals("PONG", afterFailoverJedis.ping());
-    assertEquals("foobared", afterFailoverJedis.configGet("requirepass").get(1));
+    assertNotNull(afterFailoverJedis.configGet("requirepass").get(1));
```

Comment:
Keep this unchanged. There is pending PR related to this.

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0